src/main/java/edu/harvard/iq/dataverse/ingest/IngestUtil.java

```diff
@@ -57,7 +57,7 @@
      *
      * @param version the dataset version
      * @param newFiles the list of new data files to add to it
-     * @param fileToReplace 
+     * @param fileToReplace
```

Comment:
I'm just putting this comment here near the end of the pull request because it's where I look for tests. If it's easy to add an integration test, please go ahead. I can see how this could be tricky because you've got to get the file to add onto the disk of the remote server somehow. So please don't worry about it if it's difficult. This could be worked on separately for all this related code (single file add, etc.),

Reply:
Sorry, this is not implemented in this PR.

Reply:
No worries.